### DISCUSSION TITLE
Zigbee2MQTT DRP (Disaster Recovery Pan) preparation

### DISCUSSION THREAD

**Discussion by wsw70** [Koenkk/zigbee2mqtt#8562] [General]
Zigbee2MQTT is central to my mental well-being - if it crashes plenty of things (including light switches) will stop working, my family will be mad and I will have to sleep in the garage.

This is why is want to prepare for the impossible: a Zigbee2MQTT crash.

The software part is based on backups - that one will be easier. I am mostly worried about the hardware one. Specifically, if my [zzh stick](https://electrolama.com/projects/zig-a-zig-ah/) fails, I have a spare CC2531 stick that I could plug in.

**My question: how can I retrieve the configuration I have with the zzh stick?** I am sure that it is not enough to just swap them (?)

**Comment by Koenkk** [comment]
Everything is saved and backed-up automatically in the data directory. Note that downgrading to a CC2531 with the 1.2 firmware is not possible from a CC2652 (https://www.zigbee2mqtt.io/information/FAQ.html#what-does-and-does-not-require-repairing-of-all-devices)

**Reply by wsw70** [reply]
> Everything is saved and backed-up automatically in the data directory

OK, so there is no need to change anything, just swap the sticks? 

> Note that downgrading to a CC2531 with the 1.2 firmware is not possible from a CC2652

Ah crap. This means I will have to get another CC2654 stick to just be able to swap.

Thanks for the information!

**Reply by Koenkk** [reply]
> OK, so there is no need to change anything, just swap the sticks?

yes, just swap
